ping/ping.c

```diff
@@ -595,6 +597,23 @@ main(int argc, char **argv)
 		}
 	}
 
+	if (rts.opt_version) {
+		char version_info[50];
+
+		if (rts.opt_json) {
+			sprintf(version_info, IPUTILS_VERSION("ping"));
+			// TODO: cleaner JSON version output, only version number in "version" and add map with build config
+			version_info[strcspn(version_info, "\n")] = 0;
+			construct_json(&rts, PING_JSON_STR, "version", version_info);
+			print_json_packet(&rts);
```

Comment:
This is way to ugly and we have just the version in `PACKAGE_VERSION` macro, so we should really just pass the `PACKAGE_VERSION` to the function that constructs the JSON.

Reply:
Any reason `PACKAGE_VERSION` was not used for the regular output so far? I would suggest to adopt both equally, but `PACKAGE_VERSION` returns only

```
20240905-42-g9599aaae034e+libcap: yes, IDN: yes, NLS: yes, error.h: yes, getrandom(): yes, __fpending(): yes
```


whereas `IPUTILS_VERSION("ping")` yields


```
ping from iputils 20240905-42-g2366f85cb9f3+
libcap: yes, IDN: yes, NLS: yes, error.h: yes, getrandom(): yes, __fpending(): yes
```

I think the `ping from iputils` part is not needed, but it is there so far.

Reply:
FYI `ping from iputils` is definitely needed because there are other ping implementations, e.g. busybox or inetutils. And because some distros (e.g. Debian) support all of them, it's needed to see not only the version but from which project it is.

Also it would be unrelated change (if we wanted it, it would have to be in a separate commit).

Reply:
> Any reason `PACKAGE_VERSION` was not used for the regular output so far?

`PACKAGE_VERSION` is used as part of `IPUTILS_VERSION()` it's not intended to be used separately. Please use `IPUTILS_VERSION("ping")` to fix the output.

Reply:
Thanks for the clarification! Reverted to use `IPUTILS_VERSION("ping")` in non-JSON output, but kept it for JSON output for now (if wanted in the future, the build information should probably be under a separate key).